conda_smithy/lint_recipe.py

```diff
@@ -300,7 +300,7 @@ def lintify(meta, recipe_dir=None, conda_forge=False):
         lints.append(
             "Using pinned numpy packages is a deprecated pattern.  Consider "
             "using the method outlined "
-            "[here](https://conda-forge.readthedocs.io/en/latest/meta.html#building-against-numpy)."
+            "[here](https://conda-forge.org/docs/meta.html#building-against-numpy)."
```

Comment:
This is still broken, but couldn't find a replacement.

Reply:
This is the new link.  I'll change it: http://conda-forge.org/docs/maintainer/knowledge_base.html#linking-numpy